tests/ComposerLockTest.php

```diff
@@ -12,7 +12,7 @@ class ComposerLockTest extends TestCase
      */
     public function pluginIsPackageOfTypePsalmPlugin(): void
```

Comment:
The test case name becomes misleading with these changes.

Reply:
renamed 👍